clang/test/SemaSYCL/delayed-parsing2.cpp

```diff
@@ -0,0 +1,4 @@
+// RUN: %clang %s -fsycl -c
+// REQUIRES: system-windows
+
+#include <windows.h>
```

Comment:
Generally you shouldn't use #includes of system headers in lit tests, it makes them fragile, unportable, and non-deterministic.  Please test only what you mean.

Reply:
There's another place in the tree where system includes are used, maybe basic_tests or unittests?

Reply:
Do we really need this test?
I think clang/test/Preprocessor/sycl-macro.cpp fully covers the fix.

Reply:
@bader, if you believe this test is redundant with sycl-macro.cpp, I will remove it.  Thanks.

Reply:
I agree with @erichkeane, that this test doesn't belong to clang's lit regression tests.

Probably the proper test should be a driver test checking that options are passed to the device compiler when we compile for Windows OS.